content/docs/use-cases/data-registries.md

```diff
@@ -202,7 +202,7 @@ and this will be picked up by Git:
 $ git status
 Changes not staged for commit:
 ...
-	modified:   music/songs.dvc
```

Comment:
@jorgeorpinel what are the recommended settings for the editor? do we require to keep tabs? (it can be annoying a bit)

Reply:
@mvshmakov please, check that if you replace tabs with spaces, make it at least 4 so that output looks more or less the same as in the DVC

Reply:
> what are the recommended settings for the editor?

We currently have a mix of spaces and tabs in command outputs throughout the docs I think, because people do whatever they prefer. I've advocated to just copy and paste the output exactly as it is from console, so that would typically include tabs.

Reply:
yep, agreed. But it's a little bit hard to manage- a lot of editors change them from Tabs to spaces automatically. Anyways, @mvshmakov let's keep the formatting the way it was before and create a ticket if we need to update out policies.

Reply:
Editors shouldn't change tabs inside code blocks, if they understand the format is MD. Anyway, it's a separate issue, agree!

Reply:
> @mvshmakov please, check that if you replace tabs with spaces, make it at least 4 so that output looks more or less the same as in the DVC

Yes, this is my default "indent on save" even for the first commit.

> yep, agreed. But it's a little bit hard to manage- a lot of editors change them from Tabs to spaces automatically. Anyways, @mvshmakov let's keep the formatting the way it was before and create a ticket if we need to update out policies.

That's it https://github.com/iterative/dvc.org/issues/1315.